code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/lb/LoadBalancerConfigSetListenersActionHandler.java

```diff
@@ -46,7 +54,7 @@ public HandlerResult handle(ProcessState state, ProcessInstance process) {
             // create a new set
             createNewListenerMaps(config, newListenerIds);
         }
-        return null;
+        return objectManager.reload(config);
```

Comment:
This is the only action handler thus far where you've reload the resource before returning. Do you think you should consistently reload it for all the handlers?